### ISSUE TITLE
Start on issue backlog triage

### ISSUE THREAD

**Issue by Rick-Anderson** [aspnet/AspNetCore.Docs#14803]
Use this query to find all issues that don't conform to the new triage requirements and were not created by a member of the doc team:
[Not Triaged query](https://github.com/aspnet/AspNetCore.Docs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3Adoc-bug+-label%3Adoc-enhancement+-label%3Asupport-request+-label%3Adocs-experience+-label%3Aneeds-more-info++-label%3Aproduct-feedback+-label%3Acode-of-conduct+-label%3Adoc-provided+-label%3Adoc-idea+-label%3Aproduct-question+-author%3Awadepickett+-author%3Ascottaddie+-author%3Aguardrex+-author%3Arick-anderson)  

Burndown count:
**Count on 10-3-19**: 515
**Count on 10-10-19**: 479
**Count on 10-12-19**: 298

Apply the appropriate labels are in the list above.  **DO NOT TRIAGE NEW ISSUES**. Check the date to make sure issue is at least 2 weeks old.

**Note**: For now if a triage backlog item has one of these labels, we are considering them triaged:
-label:doc-provided -label:doc-idea -label:product-question 

Wade will either get reporting to consider them triaged, or will go through and add the minimal additional label we have to use for sake of reporting.  product-question and doc-provided would be resolved/closed right away anyway so in theory those should not be a reporting problem.

**Comment by Rick-Anderson** [comment]
@wadepickett  can you review/edit/fix my query above?

**Comment by wadepickett** [comment]
@Rick-Anderson , just missing "-label:doc-experience", added that below.

is:issue is:open -label:doc-bug -label:doc-enhancement -label:support-request -label:doc-experience -label:needs-more-info -label:doc-idea -label:doc-provided -label:product-question -label:product-feedback

A second check we could do later is verify that only one of those minimum labels has been applied

**Comment by Rick-Anderson** [comment]
@wadepickett  can you fix the top
![image](https://user-images.githubusercontent.com/3605364/66173421-da0ef780-e5eb-11e9-8233-b7b18d7ad0f9.png)

**Comment by Rick-Anderson** [comment]
I didn't add `-label:doc-experience` because those should be closed immediately AND we don't have any legacy issues with that. Doesn't hurt to add it.

**Comment by Rick-Anderson** [comment]
@wadepickett  update the queries at that top and add `-assignee:wadepickett`  and the rest of the team. That will really narrow down the issues.

**Comment by Rick-Anderson** [comment]
@wadepickett  are we required to use P labels, ie, P1, P2, etc?

**Comment by wadepickett** [comment]
@Rick-Anderson, good question.  Priority labeling is not required as the minimum for an issue to be initially triaged.   Great to have them of course when we can, NOT required.

**Comment by wadepickett** [comment]
@Rick-Anderson, sorry, I thought the query was good as is so I left it as is and noted that to you.  However, looking back I realized code-of-conduct was missing.  I updated the query link.  It does not change the overall number of issues to triage anyway, but sorry I missed it just the same.

In the original description I also made a note about these three:  
-label:doc-provided -label:doc-idea -label:product-question

Currently  **if** an issue were left open with one of those labels above then reporting would consider it not triaged.  For product-question and doc-provided I would expect them to be closed anyway and not show up.  That said, I am trying to get reporting changed to exclude those. If it turns out the rest of the teams don't want to exclude them, then I will find the best equivalent minimum label to apply that they will agree on so it doesn't get mistakenly picked up in the report.  So for now, don't worry about them.
